### ISSUE TITLE
Not able to comment on a file's line - validation error

### ISSUE THREAD

**Issue by jimmy-ho** [jenkinsci/violation-comments-to-github-plugin#36]
Just started playing with the plugin and it appears to work fine if you make a general single comment, once it is used with **createSingleFileComments** it fails out with a validation error from github I believe, where the line number is missing or invalid.

`org.eclipse.egit.github.core.client.RequestException: Validation Failed (422): line required and an integer greater than zero`

If someone can review if there is some oddity going on with the library/plugin or I am missing something that is causing the plugin to misbehave?

Plugin version: 1.90
Jenkins version: 2.204.5
Pipeline DSL:
```
                                ViolationsToGitHub([
                                    commentOnlyChangedContent: true,
                                    commentTemplate: '',
                                    // Works
                                    //createCommentWithAllSingleFileComments: true,
                                    // Seems to be not working atm
                                    createSingleFileComments: true,
                                    credentialsId: tokenCredentialsId,
                                    gitHubUrl: '',
                                    minSeverity: 'ERROR',
                                    oAuth2Token: '',
                                    pullRequestId: env.CHANGE_ID,
                                    repositoryName: 'COMMENTED_OUT,
                                    repositoryOwner: 'COMMENTED_OUT,
                                    violationConfigs: [
                                        [parser: 'CHECKSTYLE', pattern: '.*/style_checkstyle\\.xml$', reporter: 'golangci-lint']
                                    ]
                                ])
```

Error in logs:
```
00:00:38.469  Line: "34" 
00:00:38.469  Position: "38" 
00:00:38.469  Comment: "**Reporter**: golangci-lint
00:00:38.469  **Rule**: errcheck
00:00:38.469  **Severity**: ERROR
00:00:38.469  **File**: pkg/canfeed/api/canfeed.go L34
00:00:38.470  
00:00:38.470  Error return value of &#96;s.logger.Log&#96; is not checked
00:00:38.470  
00:00:38.470  *<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1576867826>*"
00:00:38.470  org.eclipse.egit.github.core.client.RequestException: Validation Failed (422): line required and an integer greater than zero
00:00:38.470  	at org.eclipse.egit.github.core.client.GitHubClient.createException(GitHubClient.java:552)
00:00:38.470  	at org.eclipse.egit.github.core.client.GitHubClient.sendJson(GitHubClient.java:643)
00:00:38.470  	at org.eclipse.egit.github.core.client.GitHubClient.post(GitHubClient.java:757)
00:00:38.470  	at org.eclipse.egit.github.core.service.PullRequestService.createComment(PullRequestService.java:488)
00:00:38.470  	at se.bjurr.violations.comments.github.lib.GitHubCommentsProvider.createSingleFileComment(GitHubCommentsProvider.java:106)
00:00:38.470  	at se.bjurr.violations.comments.lib.CommentsCreator.createSingleFileComments(CommentsCreator.java:147)
00:00:38.470  	at se.bjurr.violations.comments.lib.CommentsCreator.createComments(CommentsCreator.java:71)
00:00:38.470  	at se.bjurr.violations.comments.lib.CommentsCreator.createComments(CommentsCreator.java:38)
00:00:38.470  	at se.bjurr.violations.comments.github.lib.ViolationCommentsToGitHubApi.toPullRequest(ViolationCommentsToGitHubApi.java:164)
```

**Comment by tomasbjerre** [comment]
Did you perform merge before doing analysis?
Does it happen for all violations or just this one?

**Comment by jimmy-ho** [comment]
Yes a merge was performed on the analysis, and Line 34 is exactly where the violation is located when reviewing the diff of the changes.

Unfortunately this is the only violation at the moment when I was testing the plugin behaviour.

**Comment by jimmy-ho** [comment]
Is there an easy way to see why the call is failing?
I've now tried with some other violations, still the same issue - really curious to see what value is being sent to github.

**Comment by tomasbjerre** [comment]
Probably fixed by: https://github.com/tomasbjerre/violation-comments-to-github-lib/pull/9
I will merge and release later today.

**Comment by jimmy-ho** [comment]
Oh awesome @tomasbjerre! Thanks for your response, looking forward to the fix so I can test again.

**Comment by tomasbjerre** [comment]
Should be corrected in 1.91. Releasing it now.

**Comment by jimmy-ho** [comment]
@tomasbjerre confirmed 1.91 fixes the problem, thanks to the fixer and for your quick release turnaround!
